commercetools-models/src/main/java/io/sphere/sdk/customers/commands/updateactions/ChangeAddress.java

```diff
@@ -18,22 +18,28 @@
 public final class ChangeAddress extends UpdateActionImpl<Customer> {
     private final Address address;
     private final String addressId;
+    private final String addressKey;
 
-    private ChangeAddress(final String addressId, final Address address) {
+    private ChangeAddress(final String addressId, final Address address, String addressKey) {
         super("changeAddress");
         this.address = address;
         this.addressId = addressId;
+        this.addressKey = addressKey;
     }
 
     public static ChangeAddress of(final String addressId, final Address address) {
-        return new ChangeAddress(addressId, address);
+        return new ChangeAddress(addressId, address, address.getKey());
+    }
+
+    public static ChangeAddress of(final String addressId, final String addressKey, final Address address) {
+        return new ChangeAddress(addressId, address, addressKey);
     }
 
     public static ChangeAddress ofOldAddressToNewAddress(final Address oldAddress, final Address newAddress) {
         if (oldAddress.getId() == null) {
```

Comment:
Here the check should be improved that the exception is only thrown if id **and** key are null